app/views/events/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Events</h1>
+<table>
+  <% @events.each do |event| %>
+    <tr>
+      <td><%= event.title %></td>
+      <td><%= link_to("Edit", events_path(event)) %></td>
+      <td><%= link_to("Delete", events_path) %></td>
```

Comment:
hi, muss vielleicht bei delete auch noch das argument (event) an den events_path? bzw, wenn nicht, warum nicht?

Reply:
und es klappt auch nicht mit dem loeschen, oder? bei meinem anderen projekt sieht es so aus: <td><%= link_to 'Destroy', story_path(story),
              method: :delete,
              data: { confirm: 'Are you sure?' } %></td>
...hmm.

Reply:
Thanks for your valuable thoughts on this. :100: I think the delete link should work with my newest commit. However, now there is a problem in events_controller showing that the event_params method does not work properly.

Reply:
Was fixed in 99e4dafbf35b125d7503f70fbbcdc8f2991c7e0a